js/components/time.js

```diff
@@ -60,7 +75,9 @@ Fliplet.FormBuilder.field('time', {
     }
   },
   mounted: function() {
-    var $vm = this;
+    if (Fliplet.Env.is('web') && (Fliplet.Env.get('ie11') || Fliplet.Env.get('safari'))) {
```

Comment:
Where did you read `Fliplet.Env.get('ie11')` returns something you can use?

We don't have such environment variable as far as I am aware. You must use `Modernizr` to detect the browser, am I correct @tonytlwu?

Reply:
Modernizr don't have any key to define safari